pilot/cmd/pilot-agent/main.go

```diff
@@ -26,6 +26,8 @@ import (
 	"text/template"
 	"time"
 
+	"istio.io/istio/pkg/spiffe"
```

Comment:
can we put spiffe under security/pkg/... since it's mostly about security feature.